src/components/field.jsx

```diff
@@ -24,170 +20,55 @@ function fieldCounts(dimInfo, field) {
   return str;
 }
 
-function firstFewValues(layout) {
-  const rowToText = row => `${row[0].qText || '<empty>'}`;
-
-  const selected = layout.qListObject.qDataPages[0].qMatrix.filter(row => row[0].qState === 'S' || row[0].qState === 'O');
-  const excluded = layout.qListObject.qDataPages[0].qMatrix.filter(row => row[0].qState === 'X');
-
-  let result = '';
-  if (selected.length > 0) {
-    result += `Example values:\n${selected.map(rowToText).join(', ')}\n`;
-  }
-  if (excluded.length > 0) {
-    result += `\nExample of excluded values:\n${excluded.map(rowToText).join(', ')}`;
-  }
-
-  return result;
-}
-
-export function FieldWithoutState({
-  field, fieldData, layout, showFilterbox, model,
+export default function Field({
+  layout, field, fieldData, onCancelSelection,
```

Comment:
`onCancelSelection` seems a bit out of place here since it really belongs to the filterbox?